### PR TITLE
Removing code coverage and cleanup tox

### PR DESCRIPTION
**Pull request by oke-aditya**
I don't think so these files or stuff are used let's see.

### CODE REVIEW

File: .circleci/unittest/linux/scripts/run_test.sh [hunk 1]
```diff
@@ -7,4 +7,4 @@ conda activate ./env
 
 export PYTORCH_TEST_WITH_SLOW='1'
 python -m torch.utils.collect_env
-pytest --cov=torchvision --junitxml=test-results/junit.xml -v --durations 20
+pytest -v --durations 20
```
Comment (NicolasHug) [inline .circleci/unittest/linux/scripts/run_test.sh:10]: we shouldn't remove the `--junitxml=test-results/junit.xml` part, this is what CircleCI uses to provide the "extended" test failure summary https://circleci.com/docs/2.0/collect-test-data/

Comment (oke-aditya) [reply .circleci/unittest/linux/scripts/run_test.sh:10]: Hmm, I didn't know that. I thought we dump the coverage file as xml. (Something like [this](https://github.com/oke-aditya/quickvision/blob/master/.github/workflows/ci-testing.yml#L41))

I think you are right. Let me rollback this
https://circleci.com/docs/2.0/collect-test-data/#pytest

Comment (oke-aditya) [reply .circleci/unittest/linux/scripts/run_test.sh:10]:
```suggestion
pytest --junitxml=test-results/junit.xml -v --durations 20
```

File: .circleci/unittest/windows/scripts/run_test.sh [hunk 2]
```diff
@@ -10,4 +10,4 @@ source "$this_dir/set_cuda_envs.sh"
 
 export PYTORCH_TEST_WITH_SLOW='1'
 python -m torch.utils.collect_env
-pytest --cov=torchvision --junitxml=test-results/junit.xml -v --durations 20
+pytest -v --durations 20
```
Comment (oke-aditya) [inline .circleci/unittest/windows/scripts/run_test.sh:13]:
```suggestion
pytest --junitxml=test-results/junit.xml -v --durations 20
```

**Review by oke-aditya — COMMENTED** [review]
Applying changes on the fly

**Comment by oke-aditya** [comment]
I think this should be good. cc @NicolasHug @pmeier

**Comment by oke-aditya** [comment]
Additionally we can think about cleaning up `pytest.ini`, `setup.cfg` and `pyproject.toml` to a single file. Probably just keep `pyproject.ml`. (Not sure if mypy too can be combined)

**Review by NicolasHug — COMMENTED** [review]
Thanks @oke-aditya .

I don't mind removing the `--cov` part as we don't use it (I think).

**Comment by NicolasHug** [comment]
> Additionally we can think about cleaning up pytest.ini, setup.cfg and pyproject.toml to a single file. Probably just keep pyproject.ml. (Not sure if mypy too can be combined)


I would rather keep things as is.
Any change we make about these will need to be carefully checked on the fbcode side, which has its own requirements and constraints. We wouldn't gain a tremendous value by merging all these into one file anyway, and the low benefit doesn't outweight the extra-work IMHO.

File: tox.ini [hunk 3]
```diff
@@ -1,4 +0,0 @@
-[flake8]
-max-line-length = 120
-ignore = F401,E402,F403,W503,W504
-exclude = docs/src
```
Comment (oke-aditya) [inline tox.ini:4]: Since it was there in this file I thought let's keep it

File: setup.cfg [hunk 4]
```diff
@@ -17,7 +17,7 @@ per-file-ignores =
     torchvision/models/mobilenet.py: F401, F403
     torchvision/models/quantization/mobilenet.py: F401, F403
     test/smoke_test.py: F401
-exclude = venv
+exclude = venv, docs/src
```
[the remark this answers is not in the document]
Comment (oke-aditya) [reply setup.cfg:20]: See my comment below.

File: tox.ini (hunk 3 above)
Comment (NicolasHug) [reply tox.ini:4]: This file is very outdated I think, we shouldn't trust whatever is in it. Let's please keep setup.cfg as it was

File: setup.cfg (hunk 4 above)
[the remark this answers is not in the document]
Comment (oke-aditya) [reply setup.cfg:20]:
```suggestion
exclude = venv
```

File: tox.ini (hunk 3 above)
Comment (oke-aditya) [reply tox.ini:4]: Reverted.

**Comment by oke-aditya** [comment]
I'm little bit scared :fearful: Since last time I had broken the docs while modifying such files

**Review by pmeier — APPROVED** [review]
LGTM, thanks @oke-aditya!

**Comment by oke-aditya** [comment]
CI seems all green. Hopefully this doesn't break stuff on fbcode side.

**Comment by oke-aditya** [comment]
I guess it's not breaking anything. (But take no risks, I can close it otherwise) 
 
cc @datumbox @NicolasHug

**Review by datumbox — APPROVED** [review]
LGTM, thanks!

**Comment by oke-aditya** [comment]
Aah finally, Is everything fine on fbcode side? I don't want to break something again :stuck_out_tongue:

**Pull request merged**